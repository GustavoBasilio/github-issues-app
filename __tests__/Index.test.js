/**
 * @format
 */

import 'react-native';
import React from 'react';
import Main from '../src/index';

import 'jest-styled-components'

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const main = renderer.create(<Main />).toJSON();
  expect(main).toMatchSnapshot();
});
