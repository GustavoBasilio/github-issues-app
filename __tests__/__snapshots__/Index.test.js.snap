// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  onLayout={[Function]}
  style={
    Object {
      "flex": 1,
    }
  }
>
  <View
    collapsable={false}
    onGestureHandlerEvent={[Function]}
    onGestureHandlerStateChange={[Function]}
    style={
      Object {
        "backgroundColor": "#eee",
        "flex": 1,
        "flexDirection": "column-reverse",
        "overflow": "hidden",
      }
    }
  >
    <View
      style={
        Object {
          "flex": 1,
        }
      }
    >
      <View
        pointerEvents="auto"
        style={
          Object {
            "bottom": 0,
            "left": 0,
            "opacity": 1,
            "paddingTop": 64,
            "position": "absolute",
            "right": 0,
            "top": 0,
            "transform": Array [
              Object {
                "translateX": 0,
              },
              Object {
                "translateY": 0,
              },
            ],
          }
        }
      >
        <View
          accessibilityElementsHidden={false}
          style={
            Object {
              "backgroundColor": "#fff",
              "flex": 1,
            }
          }
        >
          <View
            style={
              Array [
                Object {
                  "backgroundColor": "#EEEEEE",
                  "flexBasis": 0,
                  "flexGrow": 1,
                  "flexShrink": 1,
                  "paddingBottom": 0,
                  "paddingLeft": 15,
                  "paddingRight": 15,
                  "paddingTop": 0,
                },
              ]
            }
          >
            <View
              style={
                Array [
                  Object {
                    "borderBottomColor": "#E5E5E5",
                    "borderBottomWidth": 2,
                    "borderStyle": "solid",
                    "display": "flex",
                    "flexDirection": "row",
                    "marginBottom": 15,
                    "paddingBottom": 10,
                    "paddingLeft": 0,
                    "paddingRight": 0,
                    "paddingTop": 15,
                  },
                ]
              }
            >
              <TextInput
                allowFontScaling={true}
                autoCapitalize="none"
                autoCorrect={false}
                error={false}
                onChangeText={[Function]}
                placeholder="Adicionar novo repositório"
                placeholderTextColor="#E5E5E5"
                rejectResponderTermination={true}
                style={
                  Array [
                    Object {
                      "backgroundColor": "#FFFFFF",
                      "borderColor": "#E5E5E5",
                      "borderRadius": 6,
                      "borderStyle": "solid",
                      "borderWidth": 2,
                      "flexBasis": 0,
                      "flexGrow": 1,
                      "flexShrink": 1,
                      "fontSize": 14,
                      "paddingBottom": 5,
                      "paddingLeft": 15,
                      "paddingRight": 15,
                      "paddingTop": 5,
                    },
                  ]
                }
                underlineColorAndroid="transparent"
              />
              <TouchableOpacity
                activeOpacity={0.2}
                onPress={[Function]}
                style={
                  Array [
                    Object {
                      "justifyContent": "center",
                      "paddingLeft": 15,
                    },
                  ]
                }
              >
                <Text
                  allowFontScaling={false}
                  style={
                    Array [
                      Object {
                        "color": "#000000",
                        "fontSize": 20,
                      },
                      undefined,
                      Object {
                        "fontFamily": "FontAwesome",
                        "fontStyle": "normal",
                        "fontWeight": "normal",
                      },
                      Object {},
                    ]
                  }
                >
                  
                </Text>
              </TouchableOpacity>
            </View>
            <TouchableOpacity
              activeOpacity={0.2}
              data={Array []}
              keyExtractor={[Function]}
              keyboardShouldPersistsTaps="handled"
              onRefresh={[Function]}
              refreshing={false}
              renderItem={[Function]}
              showsVerticalScrollIndicator={false}
              style={
                Array [
                  Object {
                    "paddingTop": 10,
                  },
                ]
              }
            />
          </View>
        </View>
      </View>
    </View>
    <View
      onLayout={[Function]}
      pointerEvents="box-none"
      style={
        Object {
          "left": 0,
          "position": "absolute",
          "right": 0,
          "top": 0,
        }
      }
    >
      <View
        style={
          Object {
            "backgroundColor": "#FFF",
            "transform": Array [
              Object {
                "translateX": 0,
              },
            ],
          }
        }
      >
        <View
          onLayout={[Function]}
          pointerEvents="box-none"
          style={
            Object {
              "backgroundColor": "#FFF",
              "borderBottomColor": "#a7a7aa",
              "borderBottomWidth": 0.5,
              "height": 64,
              "paddingBottom": 0,
              "paddingLeft": 0,
              "paddingRight": 0,
              "paddingTop": 20,
            }
          }
        >
          <View
            style={
              Object {
                "flex": 1,
              }
            }
          >
            <View
              style={
                Object {
                  "bottom": 0,
                  "flexDirection": "row",
                  "left": 0,
                  "position": "absolute",
                  "right": 0,
                  "top": 0,
                }
              }
            >
              <View
                pointerEvents="box-none"
                style={
                  Object {
                    "alignItems": "center",
                    "backgroundColor": "transparent",
                    "bottom": 0,
                    "flexDirection": "row",
                    "justifyContent": "center",
                    "left": 0,
                    "opacity": 1,
                    "position": "absolute",
                    "right": 0,
                    "top": 0,
                  }
                }
              >
                <Text>
                  Git Issues
                </Text>
              </View>
            </View>
          </View>
        </View>
      </View>
    </View>
  </View>
</View>
`;
